hw2/04-background.html

```diff
@@ -6,23 +6,30 @@
     <title>04 - Background</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="04-background.js"></script>
   </head>
-  <body>
-    <main>
+  <body class="pt-5">
+    <main class="card col-md-5 col-sm-8 mx-auto px-3 pt-3">
       <h1>Exercise 04 - Background</h1>
       <p>
         The background color will change every few seconds (as indicated below).
         You can stop this behavior and you can also change how often the
         background color changes.
       </p>
-      <div>
-        <label>Background change interval (seconds): </label>
-        <input />
+      <div class="mb-2">
+        <label for="intervalInput"
+          >Background change interval (seconds):
+        </label>
+        <input type="number" id="intervalInput" value="3" />
       </div>
-      <div>
-        <input />
+      <div class="mb-2">
+        <input
+          type="button"
+          id="startStopButton"
+          class="btn btn-block text-light h6"
+        />
```

Comment:
Element input with attribute type whose value is button must have non-empty attribute value.

according to html validator.

Reply:
Thanks for pointing it out Saurav.
I have made the changes.